lib/AcqList/utils/queryUtils.js

```diff
@@ -20,6 +20,10 @@ export const buildFilterQuery = (queryParams, getSearchQuery) => {
       return getSearchQuery(filterValue, queryParams[SEARCH_INDEX_PARAMETER]);
     }
 
+    if (customFilterMap && Object.keys(customFilterMap).includes(filterKey)) {
```

Comment:
please add `{}` default value to `customFilterMap`

and you need just `(customFilterMap[filterKey])` - `customFilterMap[filterKey]` can be moved outside of `if` block

Reply:
Updated, please review. 
`customFilterMap[filterKey] can be moved outside of if block` - did you mean optional chaining `?.` or I didn't catch your point

Reply:
you get cb twice `customFilterMap[filterKey]`, you can get it before `if`

Reply:
Added const for `customFilterMap[filterKey]`